### DISCUSSION TITLE
Graph editor parse comments.

### DISCUSSION THREAD

**Discussion by nandotorterolo** [orientechnologies/orientdb#10048] [Q&A] [answered]
Hi, does anybody know how can I comment code and the parser ignore it using the Graph editor?
Many thanks in advance

![image](https://github.com/orientechnologies/orientdb/assets/8540107/203433e3-69f7-4099-9ac0-27b3118518bc)

![image](https://github.com/orientechnologies/orientdb/assets/8540107/08427a3a-719d-41f4-99be-873613841987)

**Comment by adriancoca** [comment, marked as answer]
Hi, you can comment in this way ->
/* example comment */
